### PR TITLE
[CUR-265] Switch priority order of tabs for curriculum pages

### PR DESCRIPTION
**Pull request by sonalisin**
## Description

- Updated initial link to go to units page instead of overview page
- Switched order of tabs in the curriculum header

## How to test

1. Go to https://deploy-preview-1989--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-primary/overview
2. Click on any subject/phase, where it should lead you to the sequence page first
3. You should see the order of tabs have also been switched

## Screenshots

How it used to look (delete if n/a):
Screenshot 2023-10-09 at 11 49 39

How it should now look:
Screenshot 2023-10-09 at 11 49 00

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

File: src/components/pages/CurriculumInfo/CurriculumHeader/CurriculumHeader.tsx [hunk 1]
```diff
@@ -131,18 +131,18 @@ const CurriculumHeader: FC<CurriculumHeaderPageProps> = ({
           label="Curriculum Selection"
           links={[
             {
-              label: "Overview",
-              page: "curriculum-overview",
+              label: "Unit sequence",
+              page: "curriculum-units",
               subjectPhaseSlug: subjectPhaseSlug,
-              isCurrent: tab === "overview",
+              isCurrent: tab == "units",
```
Comment (k-huggs) [inline src/components/pages/CurriculumInfo/CurriculumHeader/CurriculumHeader.tsx:137]: strict equality if possible please

**Review by k-huggs — APPROVED** [review]
1 small change but otherwise lgtm ✅

File: src/components/pages/CurriculumInfo/CurriculumHeader/CurriculumHeader.tsx (hunk 1 above)
Comment (sonalisin) [reply src/components/pages/CurriculumInfo/CurriculumHeader/CurriculumHeader.tsx:137]: thanks for spotting!!

**Review by HelenHarlow — APPROVED** [review]

**Review by rightsaidjames — APPROVED** [review]

**Pull request merged**